app/controllers/gift_requests_controller.rb

```diff
@@ -0,0 +1,41 @@
+class GiftRequestsController < ApplicationController
+  def new
+    @gift_request = GiftRequest.new
+    render "gift_request", locals: {list_id: params[:id]}
```

Comment:
Yup I did, and I thought so too when I was writing this initially, but the log file shows:

```ruby

Started GET "/list/1/gift_requests/new" for ::1 at 2016-04-10 20:59:58 -0400
Processing by GiftRequestsController#new as HTML
  Parameters: {"id"=>"1"}
{"controller"=>"gift_requests", "action"=>"new", "id"=>"1"}
  Rendered gift_requests/gift_request.slim within layouts/application (21.1ms)
Completed 200 OK in 152ms (Views: 150.5ms | ActiveRecord: 0.0ms)
```